.gitignore

```diff
@@ -7,4 +7,3 @@ spec/dummy/log/*.log
 spec/dummy/tmp/
 spec/dummy/.sass-cache
 coverage/
-Gemfile.lock
```

Comment:
una duda, si se usa como gema debería venir sin el `Gemfile.lock` o no?

Reply:
pillé [esto](https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore) en stackoverflow, que da [un ejemplo](https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore) en que recomiendan agregarlo 🤔

Reply:
Buena! estamos good entonces :)